src/util/hass-util.html

```diff
@@ -1,4 +1,5 @@
 <script src='../../bower_components/fecha/fecha.min.js'></script>
+<script src='../../build-temp/util.js'></script>
```

Comment:
Does this work with ES5 build?

Reply:
I guess it does as polymer-build will transpile for the ES5 build.

Reply:
I was under the impression that this gets transpiled by the polymer build process, but I don't have a complete understanding of all the moving pieces yet. Could definitely be wrong.

Reply:
Ah, missed your reply. 👍